convert_to_hdf5/src/convert_to_hdf5.py

```diff
@@ -188,6 +190,56 @@ def load_hdf_files(record_names, out_dir, data_info_mapping, meta_data, topic_in
                 rospy.loginfo('\t\tNo meta info')
     return hdf5_files
 
+def match_depth(hdf5_files, data_info_mapping):
+    upper_color_match_topic = 'vid/color/data/upper/matched_depth_index'
```

Comment:
This is more hard coded than might be ideal. You could search over the topics and find matching color and depth topics and any time there is a match, work with that. But this is OK for now. I doubt we will need anything more fancy in the scope of this project.